src/main/java/main/model/db/imports/BaseImporter.java

```diff
@@ -80,7 +80,10 @@ void addLogToImport(String log) throws AqualityException {
 
     private void createTestRun() throws AqualityException {
         if(testRun.getId() != null){
-            this.testRun = controllerFactory.getHandler(testRun).get(testRun, false, 1).get(0);
+            TestRunDto template = new TestRunDto();
+            template.setProject_id(testRun.getProject_id());
+            template.setId(testRun.getId());
+            testRun = controllerFactory.getHandler(template).get(template, false, 1).get(0);
```

Comment:
Is it necessary to check that size is more than 1?

Reply:
fixed